CRM/Civigiftaid/Report/Form/Contribute/GiftAid.php

```diff
@@ -224,11 +228,6 @@ public function select() {
       }
     }
 
-    $this->_columnHeaders['custom_gift_aid_amount'] = array(
```

Comment:
This column isn't the same as the other one for gift aid amount - its just that both happen to have the same title, which I agree is confusing. 

However, we need to keep this custom column, as it contains the correct value of gift aid amount for the line item. The default column contains gift aid amount for the full contribution, which might span across multiple line items. So, please bring this back.

Also, to alleviate the confusion, if you could find and replace all occurrences of `custom_gift_aid_amount` to `civicrm_line_item_gift_aid_amount` throughout this file, it would be great.